test/TEST-01-BASIC/test.sh

```diff
@@ -53,7 +53,7 @@ Description=Testsuite service
 After=multi-user.target
 
 [Service]
-ExecStart=/bin/bash -c 'set -x; systemctl --failed --no-legend --no-pager > /failed ; echo OK > /testok; while : ;do echo "testsuite service waiting for journal to move to /var/log/journal" > /dev/console ; for i in /var/log/journal/*;do [ -d "\$i" ] && echo "\$i" && break 2; done; sleep 1; done; sleep 1; exit 0;'
+ExecStart=/bin/bash -c 'set -x; systemctl --failed --no-legend --no-pager > /failed ; echo OK > /testok;'
```

Comment:
I usually have a slight preference for using `/bin/sh` instead since this should also work with non-bash shells (e.g. Debian/Ubuntu ship "dash" as a possible alternative) and there are no reasons why that wouldn't work.

The `set -x` can also be pulled outside:

```
ExecStart=/bin/sh -x -c 'systemctl --failed --no-legend --no-pager > /failed ; echo OK > /testok'
```

And last trailing semi-colon is not really necessary anymore.

Otherwise, looks great!